pkg/plugins/runtime/k8s/webhooks/testdata/validation/cli-user_create_zone-ns-mt.input.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   labels:
     kuma.io/origin: zone
     kuma.io/mesh: default
+    kuma.io/zone: zone-1
```

Comment:
Should we remove these in a couple of places? Maybe only in old test cases. I think it's important to check `kuma.io/zone` label is still optional

Reply:
done